src/jit/gentree.h

```diff
@@ -4944,34 +4942,40 @@ struct GenTreeStmt : public GenTree
         }
         else
         {
-            return gtPrev->AsStmt();
+            return gtPrev;
         }
     }
 
     GenTreeStmt(GenTree* expr, IL_OFFSETX offset)
-        : GenTree(GT_STMT, TYP_VOID)
-        , gtStmtExpr(expr)
+        : gtStmtExpr(expr)
         , gtStmtList(nullptr)
         , gtInlineContext(nullptr)
         , gtStmtILoffsx(offset)
 #ifdef DEBUG
         , gtStmtLastILoffs(BAD_IL_OFFSET)
 #endif
+        , gtNext(nullptr)
+        , gtPrev(nullptr)
+        , compilerAdded(false)
     {
-        // Statements can't have statements as part of their expression tree.
-        assert(expr->gtOper != GT_STMT);
-
-        // Set the statement to have the same costs as the top node of the tree.
-        // This is used long before costs have been assigned, so we need to copy
-        // the raw costs.
-        CopyRawCosts(expr);
     }
 
 #if DEBUGGABLE_GENTREE
-    GenTreeStmt() : GenTree(GT_STMT, TYP_VOID)
+    GenTreeStmt()
```

Comment:
I don't think the `DEBUGGABLE_GENTREE` ctor is needed anymore, it's no longer a `GenTree`.

Reply:
Fixed